aten/CMakeLists.txt

```diff
@@ -1,14 +1,17 @@
-if (BUILD_ATEN_MOBILE)
+if (BUILD_ATEN_MOBILE AND NOT BUILD_TORCH_MOBILE)
```

Comment:
With your changes `BUILD_ATEN_MOBILE` should just get replaced with `BUILD_MOBILE` (aka BUILD_TORCH_MOBILE)

Reply:
added INTERN_BUILD_ATEN_OPS for this condition